keps/sig-node/693-topology-manager/README.md

```diff
@@ -594,49 +584,294 @@ feature gate and set the CPU Manager policy to 'static'.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+No impact on running pods in the cluster. Subsequent pods that meet the requirement
+as explained above would go through admission check and aligned based on the
+configured policy.
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes, covered by node e2e tests:
+
+https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/topology_manager_test.go
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+A rollout can fail in case a bug is introduced in topology manager preventing already
+running pods from restarting or new pods to start.
+
+###### What specific metrics should inform a rollback?
+
+`topology_manager_admission_errors_total` can be used to determine the health of the
+feature and can be be used to determine if a rollback needs to be performed. It is
+worth noting that there could be valid cases where a pod is denied admission.
+
+Example of a valid error is `TopologyAffinityError` which is returned when topology
+manager is configured with `single-numa-node` but there are not enough resources
+available on a single numa node. 
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+No. Changes in behavior only affects pods meeting the conditions scheduled after the upgrade.
+Running pods will be unaffected by any change. This offers some degree of safety in both upgrade->rollback and upgrade->downgrade->upgrade scenarios.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No
+
+### Monitoring Requirements
+
+Monitor the following metrics:
+
+"topology_manager_admission_requests_total"
+"topology_manager_admission_errors_total"
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Inspect the kubelet configuration of the nodes: check feature gates and the policies
+or policies configured.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [X] Other (treat as last resort)
+  - Details: check the kubelet metric `topology_manager_admission_requests_total`
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+"topology_manager_admission_requests_total" can be used to determine if topology manager is
+performing its admission check.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [X] Metrics
+  - Metric name:
+    - topology_manager_admission_requests_total
+    - topology_manager_admission_errors_total
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+No.
+
+### Dependencies
+
+No.
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No.
+
+### Scalability
+
```

Comment:
A node-local feature does not make it no affect to scalability naturally right? Would you mind adding further info in this section? Thank you

Reply:
Sure, will do!

Reply:
LGTM